security/security-spring/src/test/java/org/cbioportal/web/DataAccessTokenControllerTest.java

```diff
@@ -30,16 +30,17 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-package org.cbioportal.weblegacy;
+package org.cbioportal.web;
```

Comment:
I meant that it can be moved to the `web` module instead of `security`, since `DataAccessTokenController.java` is in the `web` module. And you can annotate this class with `@Configuration` and move the beans from `DataAccessTokenControllerConfig.java` to here, and delete the config file. You can look at the controller tests under `web` module for some examples.